content/ADR-137-new-friend-request.md

```diff
@@ -104,7 +104,7 @@ cancelFriendshipPayload {
 ```
 cancelFriendshipConfirmationPayload{
     messageId: string,
-    friendRequestId: string
+    friendRequest: friendRequestPayload
```

Comment:
It's not a problem, but I'm wondering: why would you need us to send you the `friendRequestPayload` instead of the `friendRequestId` when the user cancels a friend request?

Reply:
we need to get the user related to the request, we dont have that kind of information in the context that we are receiving the message